nextflow_schema.json

```diff
@@ -105,6 +104,18 @@
                     "fa_icon": "fas fa-ban",
                     "hidden": true,
                     "help_text": "Do not load `igenomes.config` when running the pipeline. You may choose this option if you observe clashes between custom parameters and those supplied in `igenomes.config`."
+                },
+                "fasta_circular_target": {
+                    "type": "string",
+                    "description": "Specify the FASTA header of the target chromosome to extend (circularmapper only).",
```

Comment:
Specify in which cases "fasta_circular_target" and "fasta_mitochondrion_header" have to be supplied.

Reply:
Not sure what you mean here, this is already defined `(circularmapper only)`, and for the header  `mitochondrial-to-nuclear ratio calculation`

Reply:
Maybe just add `(optional, MTnucratio only)` to the description of `fasta_mitochondrion_header`. The actual documentation should have all the input details

Reply:
Done!